docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       context: ./project-ui
       dockerfile: ./Dockerfile
     ports:
-      - "3000:3000"
+      - "80:3000"
```

Comment:
The public URL features assumes port 80, so you need to route from external port 80 to internal (container) port 3000.